posthog/batch_exports/http.py

```diff
@@ -46,6 +46,7 @@ class Meta:
     def create(self, validated_data: dict) -> BatchExportDestination:
         """Create a BatchExportDestination."""
         team = Team.objects.get(id=self.context["team_id"])
+        # TODO should this take a team or not?
         export_destination = BatchExportDestination.objects.create(team=team, **validated_data)
```

Comment:
@hazzadous I wondered if we could update mypy 🙈 

One thing the new version is complaining about is that we're passing team here but it's not on the model. I'm not sure what the fix is...

Reply:
I guess get rid of it.